File: eTierMeeting_v2/API/DTO/Production/T2/CTB/SefetyViewModel.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace eTierV2_API.DTO.Production.T2.CTB
{
    public class SefetyViewModel
    {
        public int Month { get; set; }
        public int Year { get; set; }
        public List<EvaluetionCategory> Evaluetions { get; set; }
        public List<string> Lines { get; set; }
    }

    public class EvaluetionCategory
    {
        public string Item_ID { get; set; }
        public string Item_Name { get; set; }
        public string Item_Name_LL { get; set; }
        [Column(TypeName = "decimal(5, 2)")]
        public string Target { get; set; }
        public List<Achievement> Achievements { get; set; }
    }

    public class Achievement
    {
        public int HSE_Score_ID { get; set; }
        public string Item_ID { get; set; }
        public string Line_Sname { get; set; }
        [Column(TypeName = "decimal(5, 2)")]
        public string Score { get; set; }
        public bool IsPass { get; set; }
    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_models/production/T2/CTB/sefetyViewModel.ts
export interface SefetyViewModel {
    month: number;
    year: number;
    evaluetions: EvaluetionCategory[];
    lines: string[];
}

export interface EvaluetionCategory {
    item_ID: string;
    item_Name: string;
    item_Name_LL: string;
    target: string | null;
    achievements: Achievement[];
}

export interface Achievement {
    hsE_Score_ID: number;
    item_ID: string;
    line_Sname: string;
    score: string;
    isPass: boolean;
}